messaging-experiments.yaml

```diff
@@ -226,3 +226,106 @@
                   label: Not now
                   action:
                     navigate: true
+- id: MULTI-STAGE-ABOUTWELCOME-V1-78-BETA-V2
+  enabled: true
+  filter_expression: env.version >= '78.' && env.version < '79.' && env.channel == 'beta' && env.appinfo.appBuildID >= '20200610093716'
+  targeting: localeLanguageCode == "en" && [userId, "multi-stage-aboutwelcome-v1-78-beta-v2"]|bucketSample(0, 5000, 10000) &&
+    (isFirstStartup || 'multi-stage-aboutwelcome-v1-78-beta-v2' in activeExperiments)
+  arguments:
+    slug: bug-1644521-message-multi-stage-aboutwelcome-v1-78-beta-v2-beta-78-79
+    userFacingName: "About:Welcome MultiStage Welcome Beta"
+    experimentDocumentUrl: https://experimenter.services.mozilla.com/experiments/multi-stage-aboutwelcome-v1-78-beta-v2/
+    userFacingDescription: Study to validate launching multistage welcome flow via experiment framework.
+    isEnrollmentPaused: false
+    branches:
+      - slug: control
+        ratio: 1
+        groups:
+          - aboutwelcome
+        value: {}
+      - slug: treatment-static
+        ratio: 1
+        groups:
+          - aboutwelcome
+        value:
+          id: multi-stage-welcome
```

Comment:
Just to confirm, this `id` seems to be the only difference compared to the `treatment-dynamic` branch, is that expected?

Reply:
thats correct the only difference in treatment-dynamic branch is its id has text 'importable' in it, which code expects for a branch if want to display pre-imported topsites